<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="meem" failures="1" tests="2" name="Default test" time="25.943" errors="0" timestamp="2022-12-04T17:12:06 EST">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="runScenario" time="21.831"/>
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.971">
    <failure type="cucumber.runner.AmbiguousStepDefinitionsException" message="&amp;quot;enter username as DemoCSR&amp;quot; matches more than one step definition:
  &amp;quot;enter username as (.*)&amp;quot; in Login.enter_username_as_DemoSalesManager(String)
  &amp;quot;enter username as DemoCSR&amp;quot; in Login.enter_username_as_DemoCSR()
">
      <![CDATA[cucumber.runner.AmbiguousStepDefinitionsException: "enter username as DemoCSR" matches more than one step definition:
  "enter username as (.*)" in Login.enter_username_as_DemoSalesManager(String)
  "enter username as DemoCSR" in Login.enter_username_as_DemoCSR()

at cucumber.runner.Glue.stepDefinitionMatch(Glue.java:92)
at cucumber.runner.Runner.addTestStepsForPickleSteps(Runner.java:73)
at cucumber.runner.Runner.createTestCaseForPickle(Runner.java:63)
at cucumber.runner.Runner.runPickle(Runner.java:48)
at cucumber.api.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:57)
at cucumber.api.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
